test/sequential/test-net-connect-local-error.js

```diff
@@ -10,14 +10,16 @@ const expectedErrorCodes = ['ECONNREFUSED', 'EADDRINUSE'];
 const optionsIPv4 = {
   port: common.PORT,
   localPort: common.PORT + 1,
-  localAddress: common.localhostIPv4
+  localAddress: common.localhostIPv4,
+  family: 4
 };
 
 const optionsIPv6 = {
   host: '::1',
   port: common.PORT + 2,
   localPort: common.PORT + 3,
   localAddress: '::1',
+  family: 6
```

Comment:
```suggestion
```

Reply:
It's not, the `host` and `localAddress` are both IPv6, adding a `family: 6` here is redundant. It certainly doesn't harm to have it though, but I'm confident this is not necessary.